tests/unit/RenderTaskQueueTest.ts

```diff
@@ -0,0 +1,41 @@
+import RenderTaskQueue from '../../src/components/InvertedFlatList/BaseInvertedFlatList/RenderTaskQueue';
+
+jest.unmock('../../src/components/InvertedFlatList/BaseInvertedFlatList/RenderTaskQueue');
+
+describe('RenderTaskQueue', () => {
+    beforeEach(() => {
+        jest.useFakeTimers();
+    });
+
+    afterEach(() => {
+        jest.clearAllTimers();
+        jest.useRealTimers();
+    });
+
+    describe('notifyRenderingStateChange callback', () => {
+        it('should call notifyRenderingStateChange callback 2 times durning render process', () => {
```

Comment:
Can you please add comments to these tests in the format outlined in https://github.com/Expensify/App/tree/main/tests#documenting-tests? I think it will give these tests a lot more context to know _why_ the callback is expcted to be called twice.